test/skate_web/controllers/health_controller_test.exs

```diff
@@ -0,0 +1,19 @@
+defmodule SkateWeb.HealthControllerTest do
+  use SkateWeb.ConnCase
+
+  describe "index" do
+    test "returns 200 when gtfs loads", %{conn: conn} do
```

Comment:
This works, but isn't really the test you want. HealthController should return 200 when HeathWorker says the application is healthy, and 503 when it is not healthy. The tests for `HealthWorker.ready?/1` can make assertions about the GTFS.